modules/swagger-codegen/src/main/resources/JavaSpring/pathParams.mustache

```diff
@@ -1 +1 @@
-{{#isPathParam}}{{#useBeanValidation}}{{>beanValidationPathParams}}{{/useBeanValidation}}@ApiParam(value = "{{{description}}}"{{#required}},required=true{{/required}}{{#allowableValues}}, allowableValues = "{{#values}}{{{.}}}{{^-last}}, {{/-last}}{{#-last}}{{/-last}}{{/values}}"{{/allowableValues}}{{#defaultValue}}, defaultValue="{{{defaultValue}}}"{{/defaultValue}}) @PathVariable("{{baseName}}") {{>optionalDataType}} {{paramName}}{{/isPathParam}}
\ No newline at end of file
+{{#isPathParam}}{{#useBeanValidation}}{{>beanValidationPathParams}}{{/useBeanValidation}}@ApiParam(value = "{{{description}}}"{{#required}},required=true{{/required}}{{#allowableValues}}, allowableValues = "{{#enumVars}}{{#lambdaEscapeDoubleQuote}}{{{value}}}{{/lambdaEscapeDoubleQuote}}{{^-last}}, {{/-last}}{{#-last}}{{/-last}}{{/enumVars}}"{{/allowableValues}}{{#defaultValue}}, defaultValue="{{{defaultValue}}}"{{/defaultValue}}) @PathVariable("{{baseName}}") {{>optionalDataType}} {{paramName}}{{/isPathParam}}
```

Comment:
This is were the extra line has been added. Remove it and regenerate the samples.

Reply:
You means new line after {{/isPathParam}}?
I already done that and regenerate the samples, but new lines not removed.
Also I revert all changes in my branch and regenerate the samples, but new lines not removed. 
As you can see, new lines appears in blocks generated by other templates, not pathParams.mustache only. First of all PetApi.java using bodyParams.mustache for addPet method, but I don't  change bodyParams.mustache

Reply:
I pushed this change and some windows scripts changes, but samples still have some differences